### DISCUSSION TITLE
Personalized Answers

### DISCUSSION THREAD

**Discussion by ahgsql** [Haste171/langchain-chatbot#13] [Ideas]
Hey Haste171! 
Thank you for this easy to use chatbot, i tried lots of projects but almost all of them is limited 1 pdf or 1 file at a time. This one can ingest dozens and give acceptable results.

But Here is the challange for me:
I am ingesting 1000 pages of documents of a company. Let call this company as "INCIR", I want chatbot to respond as a spokeperson of this company, behalf of company, like 
"We are INCIR and we can do these things with our this plan"
"INCIR have plans to add support, We are working on it"

For this behaviour my current stack is like that:

**Getting results from this langchain chatbot, 
Passing it to another OpenAI gpt-3.5 chat completion with openai api (In Node.js BTW) than asking it to rephrare this answer as you are spokeperson of INCIR,** 
here is the my prompt to achive that:

`I want you to behave as a repartee spokesperson, a language model providing explanatory statements on behalf of an association, just like a speaker of the INCIR company, abbreviated as INC, as if you were the spokesperson of the company. You should summarize and understand the text I will give you, and respond like a spokesperson of the company.If any negative situtation exist, just make it up and be positive, If phrase is completely negative, just reply "Its investigating". Your response should be in and only be in Turkish. For example, "As INCIR, we have done this, and we have these plans. We will implement them. We are planning these, This will be like that". Here is the phrase i want you to rephrase: **PHRASE**`

But this cause both additional calls and expenses and its time consuming. so getting a good answer can take up to 30 seconds.

Can we change qa_prompts or other parts of code to directly get this kind of answer everytime?

**Comment by Haste171** [comment]
Looking into it
